.github/workflows/deploy.yml

```diff
@@ -25,28 +13,40 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Setup Pages
-        id: pages
-        uses: actions/configure-pages@v3
-
       - name: Use Node.js
         uses: actions/setup-node@v3
-      - run: npm ci
-      - run: npm run build --if-present
 
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v2
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Fetch data
+        run: npm run fetch
+
+      - name: Build Site
+        run: npm run build
+
+      - name: Upload Compiled Files
+        uses: actions/upload-artifact@v3
         with:
-          path: dist/
+          name: dist
+          path: ./dist
 
   deploy:
-    name: Deploy Dist Files
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
+    name: Deploy to GitHub Pages
     runs-on: ubuntu-latest
+
     needs: build
+
     steps:
+      - name: Download Compiled Files
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: ./dist
+
       - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v2
+        uses: peaceiris/actions-gh-pages@v3.9.3
```

Comment:
Any reason why you don't use upstream action?

Reply:
Was just what was there when I created the branch, I did a merge strategy `ours` on master earlier today, I saw a bunch of dependabot updates on master that updates non existing packages in this branch, so I discarded those changes. Did you maybe update that action recently? I must've missed it if that's the case

Reply:
Seems so. :smile:

https://github.com/srcery-colors/srcery-colors.github.io/commit/3ed699f4a1af16ff85b3312e84b08accf91fbb49

Reply:
oops :D I'll sort it.

Reply:
Fixed [1c533e5](https://github.com/srcery-colors/srcery-colors.github.io/pull/440/commits/1c533e50f278de2fae86ca8391ceecebe92caa06)

Used the last revision from master, just adding the fetch job

Reply:
❯ diff MASTER/.github/workflows/deploy.yml ~/src/srcery-colors.github.io/.github/workflows/deploy.yml
34a35,37
>       - run: npm fetch
>         env:
>           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}

Reply:
Sweet.  I'll look more into if it needs more improvements later.